config/locales/en.yml

```diff
@@ -35,6 +35,9 @@ en:
         firstname: "First name"
         lastname: "Last name"
         zipcode: "Shipping address postcode"
+      spree/order/bill_address:
+        address1: "Bill address (Street + House number)"
+        zipcode: "Bill address postcode"
```

Comment:
`±` I think the English default should actually say "Billing". 
"Bill" is a person's name, and I don't know his address 😛

Reply:
Ok fine ;) Thanks!

I've added another commit: 6a5acd1a23